backend/src/routes/blog.ts

```diff
@@ -75,6 +102,10 @@ blogRouter.use("/*", async (c, next) => {
   }
 });
 
+blogRouter.post("/", multerMiddleware(upload), async (c) => {
+  const prisma = new PrismaClient({
+    datasourceUrl: c.env.DATABASE_URL,
+  }).$extends(withAccelerate());
 blogRouter.post("/", async (c) => {
   const prisma = getDBInstance(c);
```

Comment:
This is also incorrect. 
```
const prisma = new PrismaClient({
    datasourceUrl: c.env.DATABASE_URL,
  }).$extends(withAccelerate());
  ```
  And
  
  ```
  const prisma = getDBInstance(c);
  ```
  are essentially doing the same thing. 
  
  Use this   `const prisma = getDBInstance(c);` instead of the long form.